src/v3/webpack.dev.config.ts

```diff
@@ -158,14 +159,16 @@ const devConfig: Configuration = mergeWithRules({
       }],
       // https://webpack.js.org/configuration/dev-server/#devserversetupmiddlewares
       setupMiddlewares(middlewares) {
-        const script = resolve(PLAYGROUND, 'mocks/server.js');
-        const watch = [resolve(PLAYGROUND, 'mocks')];
-        const env = { MOCK_SERVER_PORT, DEV_SERVER_PORT, BASE_URL: require(WIDGET_RC_JS).baseUrl };
+        if (!DISABLE_MOCK_SERVER) {
```

Comment:
nit: Prefer positive logic when possible:
```suggestion
        if (DISABLE_MOCK_SERVER) {
          return middlewares;
        }
```